runtime-config-linux.md

```diff
@@ -269,11 +350,31 @@ The following parameters can be specified to setup the block io controller:
    }
 ```
 
+#### Pids
+
+`pids` represents the cgroup subsystem `pids`.
+For more information, see [the pids cgroup man page](https://www.kernel.org/doc/Documentation/cgroups/pids.txt
+).
+
+The following paramters can be specified to setup the pids controller:
```

Comment:
“the pids controller” reads awkwardly to me.  Since we're in the PIDs section, how about just “the controller”?